server/utils.go

```diff
@@ -195,23 +204,38 @@ func credentialMethods(creds []MsgCredClient) []string {
 	return out
 }
 
+// isValidTag checks tag for validity: if tag contains a colon ':' then the prefix before the colon
+// must start with an ASCII letter and contain only ASCII letters, numbers and underscores.
+// The tag body may contain any combination of unicode letters, numbers, underscores and spaces.
+// The tag cannot start with a '+' or '-', cannot start or end with a space.
+func isValidTag(tag string) bool {
+	// Check if tag has a prefix.
+	if parts := tagPrefixRegexp.FindStringSubmatch(tag); len(parts) == 3 {
```

Comment:
Btw, why not simply `tagPrefixRegexp.MatchString(tag)`?

Reply:
done